components/builder-api-client/src/lib.rs

```diff
@@ -194,6 +196,16 @@ pub struct OriginChannelIdent {
     pub name: String,
 }
 
+#[derive(Clone, Serialize, Deserialize)]
+pub struct OriginInfoResponse {
+    pub default_package_visibility: String,
+    pub name: String,
+    #[serde(with = "json_u64")]
```

Comment:
This is probably outside the scope of this PR, but I think we can remove `json_u64` in favor of the more general [`serde_string`](https://github.com/habitat-sh/habitat/blob/41d4cb42f080866fff8b57c63c6d433f52daf667/components/core/src/util/serde_string.rs#L1).

Reply:
Hmm, it probably might make sense to investigate changing it in **all** the structs in this file:

```
17:15 $ grep "\"json_u64" components/builder-api-client/src/lib.rs | grep -v ///
    #[serde(with = "json_u64")]
    #[serde(with = "json_u64")]
    #[serde(with = "json_u64")]
    #[serde(with = "json_u64")]
    #[serde(with = "json_u64")]
    #[serde(with = "json_u64")]
    #[serde(with = "json_u64")]
✔ ~/Projects/habitat [jeremymv2/origin_info_subcommand L|…4⚑ 13]
17:16 $
```